pkg/container-collection/match_test.go

```diff
@@ -170,6 +170,268 @@ func TestSelector(t *testing.T) {
 				},
 			},
 		},
+		{
+			description: "Exclude container by name",
+			match:       false,
+			selector: &ContainerSelector{
+				K8s: K8sSelector{
+					BasicK8sMetadata: types.BasicK8sMetadata{
+						ContainerName: "!this-container",
+					},
+				},
+			},
+			container: &Container{
+				K8s: K8sMetadata{
+					BasicK8sMetadata: types.BasicK8sMetadata{
+						Namespace:     "this-namespace",
+						PodName:       "this-pod",
+						ContainerName: "this-container",
+					},
+				},
+			},
+		},
+		{
+			description: "Exclude container by name (no match)",
```

Comment:
It says "no match" but match below is true, what did I miss?

Reply:
It's negated. 
`- * - = + `

I'll rewrite the descriptions to be more refelective of the match operation